modules/jooby-apt/pom.xml

```diff
@@ -19,6 +19,12 @@
 
   <dependencies>
 
+    <dependency>
```

Comment:
As @agentgt suggested we don't need the dependency.

Reply:
About this dependency. You are right it is not necessary to successfully build/compile the module. But, the generated code might rely on it, and someone should bring it into the runtime, cause `jooby-apt` doesn't. It is some sort of a hidden dependency. I decided to highlight this moment at least this way.

We can change the scope to `test` or remove it completely and add some notes in the internal documentation, dunno which option is better.

Reply:
Let's remove it to make jooby-apt works as it is today. We can check from validator if at least `jooby-hibernate-validator` is on classpath and produces an error when not

Reply:
Actually, we still need it for tests, so, just changed the scope